packages/react-compose/src/types.ts

```diff
@@ -47,6 +47,7 @@ export type ComposeRenderFunction<T extends React.ElementType = 'div', P = {}> =
 
 export type ComposeOptions<InputProps = {}, InputStylesProps = {}, ParentStylesProps = {}> = {
   className?: string;
+  classes?: ClassDictionary;
```

Comment:
can we keep these two in sync in terms of naming?
either `class` + `classes` or `className` + `classNames`

Reply:
`className` will be removed I believe, so there'd only be a classes map.